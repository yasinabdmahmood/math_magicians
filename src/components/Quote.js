const Quote = () => (
  <div className="quote-container" data-testid="div">
    <p>
      Mathmatics is not about numbers, equation, computation, or algorithms
      It is about understanding. -- William paul Thurston
    </p>
  </div>
);
export default Quote;
