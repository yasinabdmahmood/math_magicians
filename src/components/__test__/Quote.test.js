import { render, screen, fireEvent } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from "../Quote"

it ("Check if Item component has changed", () => {
    const tree = renderer.create(
        <Quote />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});


    it('the component should render a div with class quote-container', () => {
        render(
            <Quote/>
        );
        const div = screen.getByTestId("quote-page");
        expect(div).toHaveClass('quote-container');
       
    });
   