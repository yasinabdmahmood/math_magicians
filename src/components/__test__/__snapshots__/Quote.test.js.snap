// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if Item component has changed 1`] = `
<div
  className="quote-container"
  data-testid="quote-page"
>
  <p>
    Mathmatics is not about numbers, equation, computation, or algorithms It is about understanding. -- William paul Thurston
  </p>
</div>
`;
